src/domain-services/base-types.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, ObjectType } from 'type-graphql';
+
+@ObjectType()
+export class BaseType {
+  @Field()
+  createdAt: Date;
+
+  @Field()
+  updatedAt: Date;
+}
+
+@ObjectType()
+export class ParanoidType extends BaseType {
```

Comment:
Is this concept borrowed from `sequalize`? (It's the only time I've come across soft delete being described in this way).

Personally I think that this is not descriptive naming, and would prefer something along the lines of `BaseTypeWithDeletedAt` or `BaseTypeWithSoftDelete`

Reply:
Yes, I've also seen this way of naming soft deletes in `sequelize` only. But used them anyway as I've seen https://github.com/UN-OCHA/hpc_service/pull/2241 enforce the term as well.

Renamed to `BaseTypeWithSoftDelete` which is indeed better name in 1f66d00ca0fdaf564406617dbe406f451e7279fa